### ISSUE TITLE
Suggestion : Titles can be textures

### ISSUE THREAD

**Issue by Krampus69** [YUNG-GANG/Travelers-Titles#10]
hello, :)
this mod is really cool. I was wondering if there could be a way to call pictures instead of name strings when entering a new area. Would be nice to have a nice pixel art title for dimensions,

You know something like this :
![j](https://user-images.githubusercontent.com/81943963/129982116-b83d6fb8-4179-4a53-a9e5-813af5613685.png)

But with dimension names for example. You wouldn't have to make the artworks, just let the possibility for pixel artists would be cool (of course it's not a priority, just an idea :D )
thank you for your awesome work :wave:

**Comment by Corvus20** [comment]
Reminds me of the one feature that the "Ambience - Extras" mod had in its older versions. I like this idea!

**Comment by Krampus69** [comment]
Yes exactly like this kind of stuff, that was lit ! Here is a gif for example

![giphy](https://user-images.githubusercontent.com/81943963/130259148-edb46391-d424-4630-92ea-1fb861fc6b55.gif)

Unfortunately, the 1.16.5 version doesn't have this feature. It stayed in 1.12, so yeah...Would be cool :3

**Comment by MrGreenTea** [comment]
Isn't that what visual travelers titles does?

https://www.curseforge.com/minecraft/texture-packs/visual-travelers-titles
